### ISSUE TITLE
we aim for a completed website with peppol ?

### ISSUE THREAD

**Issue by Triantafullenia-Doumani** [pondersource/peppol-python#9]
If yes, we must think about the timetable with all the stages.

**Comment by michielbdejong** [comment]
Yes! As discussed this morning. So in Django I think common practice is to add settings.py to .gitignore, so for now we can maintain settings.py separately on our own machines and then we improve the process as we learn.
